src/constants/bases.ts

```diff
@@ -0,0 +1,181 @@
+import { ChainId, Token, WETH } from '@kyberswap/ks-sdk-core'
+
+import { AMPL, COMP, DAI, MKR, USDC, USDT, WBTC_ARBITRUM } from './tokens'
+
+// a list of tokens by chain
+type ChainTokenList = {
+  readonly [chainId in ChainId]: Token[]
+}
+
+const WETH_ONLY: ChainTokenList = {
```

Comment:
all of them bold have 1 element in array, can we update to
`[ChainId.RINKEBY]: WETH[ChainId.RINKEBY],`
and remove. `...WETH_ONLY` here
<img width="368" alt="image" src="https://user-images.githubusercontent.com/33005392/201388109-caabbb3e-e161-49df-8f95-b07f2d52a4bb.png">

Reply:
It helps init default values for `BASES_TO_CHECK_TRADES_AGAINST`, `SUGGESTED_BASES` by ...WETH_ONLY